environments/puppet/hiera.yaml

```diff
@@ -23,16 +56,41 @@ hierarchy:
     - "%{facts.os.name}/%{facts.os.release.major}.yaml"
     - "%{facts.os.name}.yaml"
 
+  - name: EYAML per-hostgroup data
+    lookup_key: eyaml_lookup_key
+    paths:
+    - "hostgroups/%{hostgroup}.eyaml"
+    options:
+      pkcs7_private_key: /var/lib/puppet/keys/private_key.pkcs7.pem
+      pkcs7_public_key: /var/lib/puppet/keys/public_key.pkcs7.pem
+
   - name: Per-hostgroup data - See site.pp for more information
     paths:
-    - "hostgroups/%{::hostgroup}.yaml"
+    - "hostgroups/%{hostgroup}.yaml"
```

Comment:
This is a top-scope variable, so [it should keep the `::` prefix](https://puppet.com/docs/puppet/latest/hiera_merging.html#interpolate_puppet_variable).

```suggestion
    - "hostgroups/%{::hostgroup}.yaml"
```